mlflow/langchain/langchain_tracer.py

```diff
@@ -31,9 +32,37 @@
 VS_INDEX_DOC_URL_COL = "doc_uri"
 
 
-# Langchain config Keys
-CONFIGURABLE_KEY = "configurable"
```

Comment:
I forgot to address comment here: https://github.com/mlflow/mlflow/pull/13828#discussion_r1853379665